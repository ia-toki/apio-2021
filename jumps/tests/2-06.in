209636825ecd2c38299498cd26890ef71dd128c2
176 129
131 128 81 36 109 48 70 75 154 64 147 43 138 167 55 62 122 102 27 2 168 118 69 114 92 139 112 111 10 30 22 54 129 89 73 172 78 4 163 63 76 82 150 83 20 74 79 174 137 121 58 99 44 127 77 24 45 164 7 87 59 143 46 28 119 61 170 39 125 117 157 47 135 15 116 103 149 175 13 85 141 32 162 148 29 101 152 115 142 176 9 106 155 161 126 17 95 57 124 171 37 11 49 19 33 113 71 156 40 41 60 97 160 86 140 8 108 153 3 110 151 65 158 5 42 67 35 98 14 166 68 52 6 34 145 72 18 26 144 80 38 90 53 123 21 130 173 96 25 159 120 107 169 105 146 104 91 23 12 136 133 132 51 56 50 93 100 88 16 134 165 31 84 66 94 1
0 0 175 175
37 171 172 175
2 3 71 110
12 30 135 157
61 108 122 160
29 63 121 145
9 105 174 174
52 124 173 174
114 139 158 164
2 39 80 131
8 108 163 170
14 40 120 163
42 68 81 96
4 8 77 166
45 73 149 155
14 20 61 150
38 164 172 175
49 64 93 96
38 95 166 171
35 37 72 151
24 91 133 152
3 99 154 175
40 62 161 161
7 133 153 174
64 145 161 169
68 117 161 169
19 70 111 112
8 43 126 135
15 128 138 139
68 81 85 99
23 85 173 173
9 18 63 122
26 80 82 171
108 114 168 168
34 143 172 175
70 124 126 154
104 144 175 175
0 12 143 146
3 16 98 151
3 105 133 150
13 24 170 170
91 156 166 175
2 58 81 156
1 43 135 151
52 112 143 167
14 169 171 172
75 99 131 158
0 22 161 165
22 96 143 166
110 122 127 164
25 129 160 170
15 39 127 174
5 29 112 172
4 9 149 174
74 94 106 136
42 58 139 156
68 114 168 172
18 33 113 170
31 80 150 159
3 38 126 172
4 13 126 167
20 78 167 174
110 121 138 170
6 18 58 151
16 64 116 128
119 165 169 171
116 162 175 175
126 158 159 174
24 154 167 172
3 167 171 171
7 15 111 141
44 117 122 138
4 82 136 143
29 140 173 175
53 114 123 137
47 83 98 153
64 67 148 164
2 3 48 67
12 31 63 125
10 34 98 101
8 63 158 166
72 97 164 164
56 101 135 155
71 169 174 174
49 59 81 107
39 101 146 160
71 88 112 146
2 108 122 135
13 53 172 172
26 58 89 152
89 95 166 166
9 88 93 168
2 26 55 55
7 21 122 162
13 123 161 165
2 128 159 165
19 27 67 171
65 67 136 144
76 108 164 172
52 106 124 135
4 6 29 114
25 75 79 101
82 126 150 156
54 82 113 125
2 63 152 172
28 96 142 174
0 0 122 127
107 145 167 168
26 32 51 78
43 139 142 150
104 163 169 173
9 113 145 147
85 171 174 174
3 82 149 167
58 144 174 174
50 163 175 175
37 40 51 90
7 88 157 172
55 57 138 145
41 80 91 93
70 86 138 169
35 135 153 155
4 16 28 143
119 160 171 171
94 119 153 161
19 33 83 151
22 59 96 125
5 30 43 81
21 52 121 160
