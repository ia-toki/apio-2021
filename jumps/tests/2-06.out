fc5005cd102fffc504bc1615010ebf88823d65f3
OK
-1
1
7
-1
1
-1
-1
-1
-1
3
-1
-1
2
5
-1
3
-1
-1
-1
2
4
-1
-1
-1
-1
-1
-1
-1
-1
1
-1
4
1
-1
-1
1
-1
-1
-1
1
-1
1
2
-1
1
-1
1
-1
2
1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
2
4
-1
1
-1
1
1
-1
-1
1
-1
-1
1
-1
-1
-1
3
-1
-1
-1
1
-1
3
1
-1
2
-1
2
-1
-1
-1
-1
-1
-1
3
-1
-1
3
4
2
-1
-1
-1
2
-1
-1
3
1
1
1
2
-1
-1
-1
2
-1
-1
-1
-1
-1
2
-1
-1
4
-1
2
-1
